Flow.Launcher/Themes/Base.xaml

```diff
@@ -89,11 +91,7 @@
         <Setter Property="Minimum" Value="0" />
         <Setter Property="Visibility" Value="Visible" />
         <Setter Property="Foreground" Value="#26a0da " />
-        <Style.Triggers>
-            <DataTrigger Binding="{Binding Result.ProgressBar}" Value="{x:Null}">
-                <Setter Property="Visibility" Value="Collapsed" />
-            </DataTrigger>
-        </Style.Triggers>
+        <Setter Property="Visibility" Value="{Binding Result.IsProgressBarVisible, Converter={StaticResource BooleanToVisibilityConverter}}"/>
```

Comment:
Please help me understand why we want to put the Visibility binding to the theme file not directly in the Listbox xaml.

Reply:
original code is in base.xaml. so i keeped it that way.
But I think we can move the Visibility binding to ```ResultListBox.xaml```.

Reply:
@onesounds do you have special reason for doing this? I didn't notice this when reviewing that pr.

Reply:
ok @Sparrkle could you please move the visibility binding to `ResultListBox.xaml`?

Reply:
Okay. i have moved it.